ui/page/root/overview_page.go

```diff
@@ -622,6 +620,102 @@ func (pg *OverviewPage) recentProposal(gtx C) D {
 	})
 }
 
+func (pg *OverviewPage) calculateTotalAssetBalance() (map[libutils.AssetType]int64, error) {
+	wallets := pg.WL.AssetsManager.AllWallets()
+	assetsTotalBalance := make(map[libutils.AssetType]int64)
+
+	for _, wal := range wallets {
+		if wal.IsWatchingOnlyWallet() {
+			continue
+		}
+
+		accountsResult, err := wal.GetAccountsRaw()
+		if err != nil {
+			return nil, err
+		}
+
+		for _, account := range accountsResult.Accounts {
+			assetsTotalBalance[wal.GetAssetType()] += account.Balance.Total.ToInt()
+		}
+	}
+
+	return assetsTotalBalance, nil
+}
```

Comment:
i'm leaning towards breaking this down further and moving into components.go

calculateTotalAssetBalance()
----calculateTotalDCRBalance()
----calculateTotalBTCBalance()
----calculateTotalLTCBalance()

might makes things easier for the calculating overall total USD balance

but on second thought, since you already calculate total usd balance for each asset, might as well sum it up and display the total USD balance at the same time solving this issue https://github.com/crypto-power/cryptopower/issues/77

Reply:
I think this would be better moved in the PR for #77.
That way the changes will centre around a totally different use case.

Reply:
yes moving was the first option.

second option was to just calculate the total usd balance in the same fetchExchangeRate() method

Reply:
I will figure. out the best approach when i start working on 77.